### DISCUSSION TITLE
How to avoid unexpected rebuild that is caused by the changing of data file

### DISCUSSION THREAD

**Discussion by wangxiaochuTHU** [tauri-apps/tauri#7177] [Q&A] [answered]
Currently I use a data file to save some user specific configuration and use `.toml` as its format extension.

After I debugging for a whole night, it seems to turn out that the changing of this `.toml` file would cause the application to rebuild. 

Since I'd like to keep the `.toml` file to be its current state, how can I set my project not to detect the changing of this file? Thank you.

Currently, I use a relative path to place this file, i.e.
```
app\
    config\
        user.toml      /*  here is the .toml data file */
    src\
        App.jsx
        ....
    src-tauri\
        src\
            main.rs
            ....
```

**Comment by FabianLars** [comment, marked as answer]
try adding a `.taurignore` file next to tauri.conf.json with something like
```
user.toml
```
or
```
**/user.toml
```
(the syntax/logic _should_ be really similar to classic .gitignore giles)

**Reply by wangxiaochuTHU** [reply]
I works fine, thanks for your reply!
